### PR TITLE
feat(dev-console): allow devconsole to work on native k8s

### PR DESCRIPTION
**Pull request by abhinandan13jan**
Sets `/add` as `landingPage` and hides `topology`navlink when Openshift flag is false.
To test, manually override `Openshift` flag to false from `features.ts`

### CODE REVIEW

File: frontend/public/components/app-contents.tsx [hunk 1]
```diff
@@ -101,128 +100,548 @@ const pageRouteExtensions = plugins.registry.getRoutePages().map((r) => {
...
-
-          <Route path="/k8s/all-namespaces/:plural" exact component={ResourceListPage} />
-          <Route path="/k8s/all-namespaces/:plural/:name" component={ResourceDetailsPage} />
-
-          <LazyRoute path="/error" exact loader={() => import('./error' /* webpackChunkName: "error" */).then(m => m.ErrorPage)} />
-          <Route path="/" exact component={DefaultPage} />
-
-          <LazyRoute loader={() => import('./error' /* webpackChunkName: "error" */).then(m => m.ErrorPage404)} />
-        </Switch>
+const AppContents = withRouter(
```
Comment (spadgett) [inline frontend/public/components/app-contents.tsx]: I'd like to enable prettier for the entire repo, but in the meantime running it on this file makes the PR difficult to review. It's unclear what's changed here.

**Comment by christianvogt** [comment]
@abhinandan13jan Looks like the original solution of a single `landingPageURL` isn't sufficient for this use case. 

You haven't addressed the redirect in `app-contents#DefaultPage_`.

We might be better off keeping the default landing page URL of all perspectives a constant set to `/` and use `Page/Route` extension to allow perspectives to override the route for `/`. We'd need to augment `Page/Route` with a `perspective` property. (Note: the addition of perspective to `Page/Route` is something that we already need for another feature)

If we did this, then dev console could contribute a `Page/Route` for `/` which checks the flags and redirects to the appropriate landing page. It would be similar logic to what's already in `app-contents#DefaultPage_` for admin perspective.

File: frontend/public/components/app-contents.tsx (hunk 1 above)
Comment (christianvogt) [reply frontend/public/components/app-contents.tsx]: Agreed. We should not be reformatting entire files.

**Comment by christianvogt** [comment]
@abhinandan13jan here's what I was thinking for the Page/Route perspective enhancement:
https://github.com/christianvogt/console/commit/e305bd6e8f76004090da33f036c090089d3bac96

File: frontend/public/components/app-contents.tsx (hunk 1 above)
Comment (abhinandan13jan) [reply frontend/public/components/app-contents.tsx]: My bad. I din't know until raising the PR that the file wasn't prettified. Anyway, I'll redo the changes with the common landing Page

**Pull request closed without merging**